src/contexts/environment-context.js

```diff
@@ -30,20 +29,25 @@ export const EnvironmentProvider = ({ children }) => {
     }
   }
 
-  // store csrf token
-  const [csrfToken, setCsrfToken] = useState();
+  // load session timeout value
+  const loadTimeout = async () => {
+    const user_response = await userHandler(helxAppstoreUrl);
+    setTimeoutSeconds(user_response.data['SESSION_TIMEOUT']);
+  }
 
   useEffect(() => {
     loadContext();
     loadCsrfToken();
+    loadTimeout();
   }, [])
 
   return (
     <EnvironmentContext.Provider value={{
       helxSearchUrl: process.env.REACT_APP_HELX_SEARCH_URL,
-      helxAppstoreUrl: window.location.origin,
+      helxAppstoreUrl: helxAppstoreUrl,
       context: context,
-      helxAppstoreCsrfToken: csrfToken
+      helxAppstoreCsrfToken: csrfToken,
+      timeoutSeconds: 200
```

Comment:
Should this be the value from the user data minus 100 or a percentage of that timeout instead of a hard coded value? Could be annoying it a timeout is set to 20 minutes, but we pop this up more often.

Reply:
Right, timeout value from /user is loaded and will be consumed. I set this for local testing purpose, sorry for the confusion!